Package.swift

```diff
@@ -0,0 +1,27 @@
+// swift-tools-version:5.1
+import PackageDescription
+
+let package = Package(
+    name: "UILabel-Copyable",
```

Comment:
It would be better to keep the package name the same as the Pod/Library: `UILabel+Copyable`
```suggestion
    name: "UILabel+Copyable",
```